db/migrations/20230208050719_product_feelings.sql

```diff
@@ -0,0 +1,8 @@
+-- migrate:up
+CREATE TABLE product_feelings(
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    product_feeling VARCHAR(20)
```

Comment:
컬럼명이 name 혹은 type은 어떨까요?
```suggestion
    type VARCHAR(20)
```